install/install.sh

```diff
@@ -53,11 +53,43 @@ then
   usage;
 fi
 
+if grep -q -i "release 6" /etc/redhat-release || grep -q -i "release 7" /etc/redhat-release
+then
+  install_with_yum=1
+elif grep -q -i "release 7" /etc/centos-release || grep -q -i "release 6" /etc/centos-release
+then
+  install_with_yum=1
+elif grep -q -i "release 28" /etc/fedora-release || grep -q -i "release 27" /etc/fedora-release
+then
+  install_with_dnf=1
+else
+  echo "Installation failed. Could not find a supported Operating System."
```

Comment:
If you change the package manager logic you can check if this is a redhat OS by running:

```sh

if [ ! -f /etc/redhat-release ]; then
	echo "/etc/redhat-release not found. You need to run this on a Red Hat based OS"
fi